From ea4d256808dbf01bcd3bc79e52d57be21e6ad173 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Thu, 21 Nov 2024 12:08:01 -0500
Subject: [PATCH] Downstream-only: Allow passing CMake defs through an env.
 var.

It is too hard to get the build_ext command-line arguments passed
through when building a wheel.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 03d6161..56a4b87 100644
--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,7 @@ class CMakeBuild(build_ext):
 
     def initialize_options(self):
         build_ext.initialize_options(self)
-        self.cmake_defs = None
+        self.cmake_defs = os.environ.get('MORPHIO_CMAKE_DEFS')
 
     def run(self):
         for ext in self.extensions:
-- 
2.47.0

